content/backend/release/mainapp.yaml.tpl

```diff
@@ -1,30 +1,30 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: nameLower-mainapp
+  name: lncdapp-mainapp
   namespace: ${NAMESPACE}
   labels:
-    app: nameLower
+    app: lncdapp
     component: mainapp
     environment: ${ENVIRONMENT}
 spec:
   selector:
     matchLabels:
-      app: nameLower
+      app: lncdapp
       component: mainapp
       environment: ${ENVIRONMENT}
   replicas: 1
   revisionHistoryLimit: 3
   template:
     metadata:
       labels:
-        app: nameLower
+        app: lncdapp
         component: mainapp
         environment: ${ENVIRONMENT}
```

Comment:
Let's set `aadpodidbinding` here and in migrations, shouldn't hurt anyone if unused but one less thing to remember when configuring managed identities.